BaseFilter/sections/antiadblock.txt

```diff
@@ -741,7 +741,7 @@ financeyogi.net,onroid.com,sportfacts.net,youboxtv.com,itdmusics.com#%#//scriptl
 ! #%#//scriptlet('prevent-eval-if', 'adbEnableForPage')
 migdalor-news.co.il,gtf.com.ng,xn--nbkw38mlu2a.com,yuki0918kw.com,bcanotesnepal.com,conoscereilrischioclinico.it,nvimfreak.com,openstartup.tm,stablediffusionxl.com,techsbucket.com,f1ingenerale.com,maketoss.com,iwb.jp,certificateland.com,downloadtanku.org,limontorrent.com#%#//scriptlet('prevent-eval-if', 'adbEnableForPage')
 ! #%#//scriptlet('set-constant', 'adbEnableForPage', 'false')
-rshostinginfo.com,splashbase.co,ralli.ee,camcam.cc,nsw2u.*,mundovideoshd.com,ayatoon.com,sapphirescan.com,noblessetranslations.com,eventiavversinews.it,studybullet.com,e-kitapstore.com,studybullet.com,zapcourses.com#%#//scriptlet('set-constant', 'adbEnableForPage', 'false')
+3dpchip.com,rshostinginfo.com,splashbase.co,ralli.ee,camcam.cc,nsw2u.*,mundovideoshd.com,ayatoon.com,sapphirescan.com,noblessetranslations.com,eventiavversinews.it,studybullet.com,e-kitapstore.com,studybullet.com,zapcourses.com#%#//scriptlet('set-constant', 'adbEnableForPage', 'false')

```

Comment:
We also need rules for AdGuard, which does not support scriptlets.
exceptions can be added under this directive in this section:
`!#if (adguard_ext_android_cb)`
I got something like:
```
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=3dpchip.com
@@||ads.pinterest.com^$domain=3dpchip.com
@@||sovrn.com^$domain=3dpchip.com
```

Reply:
@jellizaveta Thx, updated